.eslintrc

```diff
@@ -34,6 +34,7 @@
       "files": ["*.ts", "*.tsx"],
       "extends": ["plugin:@typescript-eslint/recommended"],
       "rules": {
+        "@typescript-eslint/explicit-function-return-type": "off",
```

Comment:
for some reason I kept getting warnings that the component functions needed return types :(